instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-jersey-2.0/jaxrs-2.0-jersey-2.0.gradle

```diff
@@ -24,3 +24,7 @@ dependencies {
   testImplementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.2.3'
   testImplementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-afterburner', version: '2.9.10'
 }
+
+test {
+  systemProperty 'testLatestDeps', testLatestDeps
```

Comment:
Why is this needed? @anuraaga is it?

Reply:
A, got it...

Reply:
A, but this is a great point out. @mateuszrzeszutek if you get a chance to add a one line comment here sometime to avoid future confusion would be great!